variables.tf

```diff
@@ -59,6 +62,23 @@ variable "server_cert_crn" {
   description = "CRN of a secret in Secrets Manager that contains the certificate to use for the VPN"
 }
 
+variable "client_cert_crns" {
+  type        = list(string)
+  description = "List of client CRN certificates used for VPN authentication."
```

Comment:
This desrption should mention that the list cannot be empty if `client_auth_methods` value contains `certificate` value.

Reply:
same update needed in the DA